### PR TITLE
pkp/pkp-lib#690: user login block should not expose a form via http if force_login_ssl is set

### PR DESCRIPTION
**Pull request by ctgraham**
See https://github.com/pkp/pkp-lib/issues/690

Also, move hardcoded English string to locale key.

### CODE REVIEW

File: plugins/blocks/user/UserBlockPlugin.inc.php [hunk 1]
```diff
@@ -61,9 +61,16 @@ function getContents(&$templateMgr) {
 			$templateMgr->assign_by_ref('userSession', $session);
 			$templateMgr->assign('loggedInUsername', $session->getSessionVar('username'));
 			$loginUrl = Request::url(null, 'login', 'signIn');
+			// if the page is not ssl enabled, and force_login_ssl is set, this flag will present a link instead of the form
+			$forceSSL = 0;
```
Comment (asmecher) [inline plugins/blocks/user/UserBlockPlugin.inc.php]: Looks good, but do you mind changing the flag to use true/false rather than numbers?

**Comment by ctgraham** [comment]
Changed flag: int 0/1 to boolean false/true.

File: plugins/blocks/user/block.tpl [hunk 2]
```diff
@@ -28,7 +28,9 @@
 		</ul>
 	{else}
 		{if $implicitAuth}
-			<a href="{url page="login" op="implicitAuthLogin"}">Journals Login</a>
```
Comment (asmecher) [inline plugins/blocks/user/block.tpl]: Good catch on this too.

**Comment by asmecher** [comment]
Thanks, @ctgraham -- merged, and I'll also cherry-pick it to ojs-stable-2_4_6.

**Pull request merged**